Automattic/wp-calypso
# Update/home add growth summit task

This PR adds the client-side code for a new My Home card that promotes the upcoming Growth Summit. Additional context for this request is at pbvYe0-5F-p2. This won't go live until the backend code is deployed. This task card will only display to English-speaking users because the conference is English-only.

Here's a screenshot of the current design:
<img width="1030" alt="Screen Shot 2020-07-16 at 10 07 44 PM" src="https://user-images.githubusercontent.com/35781181/87740695-d3372700-c7b0-11ea-8a48-15d5947cc10e.png">

The design and copy are still in progress and will be refined.

#### Testing instructions

* Checkout the branch (git checkout update/home-add-growth-summit-task)
* Patch your sandbox to D46543-code
* Update your hosts file to point API calls to your sandbox
* Confirm that your API calls are sandboxed (https://public-api.wordpress.com/?amisandboxed)
* Force the Growth Summit view to display on My Home (http://calypso.localhost:3000/home/SITESLUG?dev=true&view=VIEW_GROWTH_SUMMIT&flags=-home/layout-dev)
